### DISCUSSION TITLE
DockerFile

### DISCUSSION THREAD

**Discussion by andrealmei** [directus/directus#4581] [Archive]
Hi,
I got a standalone directus to start learn and work.
How can I build docker with this? Someone have a Dockerfile and instructions to create this?
Tks.

**Comment by juancarlosjr97** [comment]
Hi, 

Have a look here: https://docs.directus.io/guides/installation/docker/

The example is a docker-compose, but if you need a `Dockerfile`, you can copy the image and adapt it to a Dockerfile.

This a Dockerfile I used for when the first release candidates.

```
FROM directus/next:v9.0.0-rc.3

ENV PORT=80
ENV PUBLIC_URL="/"

ENV RATE_LIMITER_ENABLED=false
ENV RATE_LIMITER_STORE=memory
ENV RATE_LIMITER_POINTS=25
ENV RATE_LIMITER_DURATION=1

ENV CACHE_ENABLED=false

ENV ACCESS_TOKEN_TTL="15m"
ENV REFRESH_TOKEN_TTL="7d"
ENV REFRESH_TOKEN_COOKIE_SECURE=false
ENV REFRESH_TOKEN_COOKIE_SAME_SITE="lax"

ENV OAUTH_PROVIDERS=""

ENV KEY="5957be82-8c7d-45c0-9251-a23879224067"
ENV SECRET="lQtLbrovvvCpsx_ADh0qInKYxPUOW93z"

EXPOSE 80
```

**Reply by andrealmei** [reply]
Tks for your answer, but I'd like to create a new image from my source code.

In other words I'd like to run something line this:

git clone https://github.com/directus/directus.git
docker build -t directus_img .

**Reply by juancarlosjr97** [reply]
You can have a look to the build on the repository of how the image of Directus is built: https://github.com/directus/directus/tree/main/.github/actions/build-images

**Reply by andrealmei** [reply]
Tks Juan, but I cant't understand what files I need to put on rootfs.
Could you explain a bit more how this works? There are any docs with this?
When I run the project build I got two compiled folders, right?
- app/dist
- api/dist
My problem is understand where I put this files to build docker and then how to start this.
